src/views/Workflow.vue

```diff
@@ -54,14 +66,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <script>
 import { mixin } from '@/mixins'
 import { datatree } from '@/mixins/treeview'
+import { datatable } from '@/mixins/tableview'
```

Comment:
This can be removed.

Reply:
Done

Reply:
I also had to remove `datatable` from `views/Workflow.vue` (line 88 - in `export default` as once I removed the `import` statement, `datatable` became undefined.